Assets/LeapMotionModules/Hands/Shaders/UnLitAlpha.shader

```diff
@@ -0,0 +1,66 @@
+﻿// Unlit alpha-blended shader.
```

Comment:
What is this shader used for?  It seems to be a slightly different shader than Unlit/Transparent.